.github/workflows/assets_validation.yml

```diff
@@ -24,8 +24,10 @@ jobs:
     - uses: actions/checkout@v4
     - name: Rebuild packages
       run: |
-        npm install
-        gulp rebuild
+        npm install -g corepack
+        corepack enable
+        yarn install
```

Comment:
Shouldn't this be a `devDependency` in a `package.json`? That way, Renovate could update it, and we'd use an explicit version too. This can break our build if the version changes.

Reply:
No it is not. This is a requirement to install corepack as a global dependency so that we can decide which version of yarn we will use.

Reply:
See root package.json file 

"packageManager": "yarn@4.6.0",

Reply:
Why do we need an explicit `yarn install` here, though? Shouldn't that be handled by corepack?